src/components/application_manager/src/resumption/resumption_data_processor_impl.cc

```diff
@@ -627,9 +627,9 @@ void ResumptionDataProcessorImpl::DeleteCommands(
   };
 
   auto accessor = application->commands_map();
-  const auto& commands_map = accessor.GetData();
+  const auto commands_map = accessor.GetData();
 
-  for (const auto& cmd : commands_map) {
+  for (const auto cmd : commands_map) {
```

Comment:
```suggestion
  for (const auto& cmd : commands_map) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e